dump/src/lib.rs

```diff
@@ -736,18 +736,14 @@ where
     let values = values.into_iter();
     let len = values.len();
     let mut pieces = values.take(64).map(|piece| {
+        let mut piece = piece.to_string();
+        piece.retain(|c| !c.is_control());
```

Comment:
It would be a bit too surprising to find a mismatching number of bytes and text length. Replacing the character would be preferable over removing it entirely.
```suggestion
        piece.replace(|c| c.is_control(), '�');
```
Later on, this could be extended with more options such as one for using DICOM-compliant character codes.

Reply:
I didn't find a nice way to keep the string length. Replacement with "UNKNOWN" makes sense to me 👍